widgets/src/Project/Page.jsx

```diff
@@ -1,6 +1,14 @@
 const ownerId = "contribut3.near";
 const accountId = props.accountId ?? context.accountId;
 
+const isAdmin = Near.view(
+  ownerId,
+  "check_is_project_admin",
+  { project_id: accountId, account_id: context.accountId },
+  "final"
+);
+
+
```

Comment:
Let's use the `State.update` pattern for this like so:
```javascript
State.init({
  isAdmin: false,
  isAdminIsFetched: false,
});

if (!state.isAdminIsFetched) {
  Near.asyncView(
    ownerId,
    "check_is_project_admin",
    { project_id: accountId, account_id: context.accountId },
    "final",
    false,
  ).then((isAdmin) => State.update({ isAdmin, isAdminIsFetched: true }));
}
```

Reply:
Done. Thanks.